### DISCUSSION TITLE
Vpn interfering with sync

### DISCUSSION THREAD

**Discussion by Ianh89** [Chia-Network/chia-blockchain#3524] [Support]
Hi. I've port forwarded to 8444 on my router which seems to help when I'm not connected via vpn but as soon as I connect to it I'm constantly stuck at 'not synced' and can't connect to other peers easily either. The port is forwarding correctly when not connected to vpn but isn't when connected. I've had it working before with vpn so wondering if I'll have to stay off it or it will just sort itself out? any help would be appreciated

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
